core/runtime/src/main/java/io/quarkus/runtime/configuration/AbstractDelegatingConfigSource.java

```diff
@@ -12,9 +13,10 @@
 /**
  * A base class for configuration sources which delegate to other configuration sources.
  */
-public abstract class AbstractDelegatingConfigSource implements ConfigSource {
+public abstract class AbstractDelegatingConfigSource implements ConfigSource, Serializable {
+    private static final long serialVersionUID = -6636734120743034580L;
     protected final ConfigSource delegate;
-    private Map<String, String> propertiesMap;
+    protected Map<String, String> propertiesMap;
```

Comment:
What is the purpose of this change? Nothing new seems to use this field.

Reply:
It might have been a carryover change from when I was trying to get serialization to work and needed access to it in the children.

Can revert back if needed

Reply:
If you wouldn't mind, that would be good.

Reply:
done